package-lock.json

```diff
@@ -15864,8 +15864,7 @@
 				"ansi-regex": {
 					"version": "2.1.1",
 					"bundled": true,
-					"dev": true,
-					"optional": true
+					"dev": true
```

Comment:
There are some unrelated changes included in this file. I bet they should be removed from this patch.

Reply:
This is the result of `npm run lint-js:fix` command on my branch. Should I removed them with a new commit?

Reply:
It's rather from `npm install` :)
Yes, please remove it.

Reply:
Thanks @gziolo, we should be good to go now :)